<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
background-color:white;
border-style: solid;

border-width: 1px;
border-radius: 7px;
}
QMainWindow{
	background-color: #37474f;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="display_image">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-color: #00838f ;
border-width: 3px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="display_image_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>70</y>
      <width>351</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-color: #00838f ;
border-width: 3px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startVideo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>131</width>
      <height>54</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#eeeeee;
color: black;
border-color: white;</string>
    </property>
    <property name="text">
     <string>Start Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="joystick_status">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #64dd17; 

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="atMega_status">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>560</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #c62828;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gradient">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>61</width>
      <height>57</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0.784, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 71, 76, 255));

border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gradient_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>570</y>
      <width>61</width>
      <height>57</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0.784, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 71, 79, 255));

border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="error_video">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gradient_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0.784, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 71, 76, 255));

border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="auto_drive">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>560</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #00838f ; 

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shapes_recognize">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>560</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #00838f ; 

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gradient_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>560</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0.784, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 71, 76, 255));

border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gradient_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>560</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0.784, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 71, 76, 255));

border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gradient_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>560</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0.784, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 71, 76, 255));

border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="home">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>560</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #00838f ; 

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="trackbar_circle">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>120</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberTrackbar">
    <property name="geometry">
     <rect>
      <x>1233</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="logoLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>-90</y>
      <width>691</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cannon_measure">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>560</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #00838f ; 

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gradient_7">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>560</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0.784, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 71, 76, 255));

border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gradient_8">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>390</y>
      <width>50</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0.784, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 71, 76, 255));

border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="term_sensor">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>380</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #00838f ; 

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="console">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>560</y>
      <width>521</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-color: #00838f ;
border-width: 1px;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; color:#204a87;&quot;&gt;-- PoliTOcean Carassius v2.0:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="home_3">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #00838f ; 

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gradient_9">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>320</y>
      <width>50</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0.784, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 71, 76, 255));

border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="measure_button">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>460</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #00838f ; 

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gradient_10">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>460</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0.784, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 71, 76, 255));

border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gridLabel">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>240</y>
      <width>400</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="trackbar_circle_2">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>190</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberTrackbar_2">
    <property name="geometry">
     <rect>
      <x>1233</x>
      <y>200</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberTrackbar_3">
    <property name="geometry">
     <rect>
      <x>1233</x>
      <y>270</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="trackbar_circle_3">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>260</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="debugCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug</string>
    </property>
   </widget>
   <zorder>gridLabel</zorder>
   <zorder>display_image</zorder>
   <zorder>display_image_2</zorder>
   <zorder>gradient</zorder>
   <zorder>joystick_status</zorder>
   <zorder>gradient_2</zorder>
   <zorder>atMega_status</zorder>
   <zorder>error_video</zorder>
   <zorder>gradient_3</zorder>
   <zorder>startVideo</zorder>
   <zorder>gradient_4</zorder>
   <zorder>gradient_5</zorder>
   <zorder>shapes_recognize</zorder>
   <zorder>auto_drive</zorder>
   <zorder>gradient_6</zorder>
   <zorder>home</zorder>
   <zorder>lcdNumberTrackbar</zorder>
   <zorder>gradient_7</zorder>
   <zorder>cannon_measure</zorder>
   <zorder>gradient_8</zorder>
   <zorder>term_sensor</zorder>
   <zorder>trackbar_circle</zorder>
   <zorder>gradient_9</zorder>
   <zorder>home_3</zorder>
   <zorder>console</zorder>
   <zorder>gradient_10</zorder>
   <zorder>measure_button</zorder>
   <zorder>trackbar_circle_2</zorder>
   <zorder>lcdNumberTrackbar_2</zorder>
   <zorder>lcdNumberTrackbar_3</zorder>
   <zorder>trackbar_circle_3</zorder>
   <zorder>debugCheck</zorder>
   <zorder>logoLabel</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
